clevertap-core/src/main/java/com/clevertap/android/sdk/ManifestInfo.java

```diff
@@ -90,12 +93,28 @@ private ManifestInfo(Context context) {
 
         xiaomiAppKey = _getManifestStringValueForKey(metaData, Constants.LABEL_XIAOMI_APP_KEY);
         xiaomiAppID = _getManifestStringValueForKey(metaData, Constants.LABEL_XIAOMI_APP_ID);
+
+        profileKeys = parseProfileKeys(metaData);
+    }
+
+    public String getAccountId() {
+        return accountId;
     }
 
     public String getFCMSenderId() {
         return fcmSenderId;
     }
 
+    public String[] getProfileKeys() {
+        return profileKeys;
+    }
+
+    private String[] parseProfileKeys(final Bundle metaData) {
+        String profileKeyString = _getManifestStringValueForKey(metaData, Constants.CLEVERTAP_IDENTIFIER);
```

Comment:
`_getManifestStringValueForKey` returns NULL when the code flow goes into `catch`. Because of that we need to add a null check in the next line

Reply:
check is added to prevent null check